crates/motionfile/src/condition.rs

```diff
@@ -1,18 +1,25 @@
 use std::{fmt::Debug, time::Duration};
 
-use crate::StabilizedCondition;
+use crate::{FallenAbort, StabilizedCondition};
 
 use enum_dispatch::enum_dispatch;
 use serde::{Deserialize, Serialize};
 use types::ConditionInput;
 
+pub enum Response {
+    Abort,
+    Continue,
+    Wait,
+}
+
 #[enum_dispatch(ConditionType)]
 pub trait Condition: Clone {
```

Comment:
Do you need to require `Clone` here?

Reply:
I thought so, but apparently it works without it. Removed now.